Cargo.lock

```diff
@@ -1,5 +1,7 @@
 # This file is automatically @generated by Cargo.
 # It is not intended for manual editing.
+version = 3
```

Comment:
this should not be included I believe

Reply:
Previously we didn't want to include it but I'm good with including it if everyone agrees.

Reply:
I'm surprised it is added now. Why, if it is automatic, wasn't it added before ?

Reply:
Because I updated my rust toolchain